src/stores/pacbioRunCreate.js

```diff
@@ -508,5 +515,73 @@ export const usePacbioRunCreateStore = defineStore('pacbioRunCreate', {
       this.$reset()
       this.resources = resources
     },
+    /**
+     * Returns the available volume for a library aliquot
+     * @param {Object} libraryId The id of the library
+     * @param {Object} aliquotId The id of the aliquot
+     * @param {Object} volume The volume of the aliquot
+     *
+     * @returns {Number} The available volume for the aliquot
+     */
+    getAvailableVolumeForLibraryAliquot({ libraryId = null, aliquotId = null, volume = null }) {
+      if (!libraryId) {
+        return null
+      }
+
+      // Get the original aliquot if it exists
+      const original_aliquot = this.aliquots[aliquotId]
+      // Get the available volume for the library
+      const library_available_volume = this.libraries[libraryId].available_volume || 0
+
+      // Calculate the sum of the volume of all the new aliquots used in wells that are from the library
+      let library_used_aliquots_volume = 0
+      // eslint-disable-next-line no-unused-vars
+      Object.entries(this.wells).forEach(([_plateNumber, plate]) => {
+        // eslint-disable-next-line no-unused-vars
+        Object.entries(plate).forEach(([_position, well]) => {
```

Comment:
Would it be possible to  use Object.values instead as we are only interested those and also avoids eslint complains as well?